### DISCUSSION TITLE
Enable email scheduling

### DISCUSSION THREAD

**Discussion by VeraStray** [kewisch/gdata-provider#292] [Support] [answered]
For some reason I am unable to enable email scheduling. I can check the box, but the selection does not persist. If i leave and come back, it is not checked any more. I am trying to solve the issue: No writable calendars are configured for invitations, please check the calendar properties.

**Comment by MartinSchneiderGermany** [comment]
Same problem with me

**Comment by kewisch** [comment]
Do you get any error console messages? https://github.com/kewisch/gdata-provider/wiki/FAQ#enabling-debugging

**Reply by VeraStray** [reply]
Thanks for the reply. Yes, I do get an error message: 
When I go to the extensions / Provider for Google Calendar and then try to put a checkmark in "Enable email scheduling" the follow error comes up in the error Console: 

```
Error: Can't find profile directory. XULStore.jsm:66:15
    load resource://gre/modules/XULStore.jsm:66
    XULStore resource://gre/modules/XULStore.jsm:24
```

Screenshot attached
![Screenshot 2021-11-22 112358](https://user-images.githubusercontent.com/66409569/142906998-9ed44382-c3be-4651-ae53-ab7636a2bc60.png)
.
My profile directory is: 
C:\Users\xxxxxx\AppData\Roaming\Thunderbird\Profiles\xxxxxxx

Some other errors on Thunderbird Startup:
![image](https://user-images.githubusercontent.com/66409569/142907797-e174c855-ec44-42bd-8dd6-708a9ab6758b.png)

![image](https://user-images.githubusercontent.com/66409569/142907863-ee8a32f8-ff22-43a1-a90f-79467eb91a80.png)

![image](https://user-images.githubusercontent.com/66409569/142907968-d5d3ef61-3533-4479-a6c5-03b631a7cb2b.png)

**Comment by kewisch** [comment, marked as answer]
Thank you for noting this, I've fixed this in #312 and will release soon.
